README.md

```diff
@@ -1,2 +1,175 @@
+
+NPR's API is a content API, which essentially provides a structured way for other computer applications to get NPR stories in a predictable, flexible and powerful way. The content that is available includes audio from most NPR programs dating back to 1995 as well as text, images and other web-only content from NPR and NPR member stations. This archive consists of over 250,000 stories that are grouped into more than 5,000 different aggregations.
+
+Access to the NPR API requires an API Key which you can get by [registering for an NPR account](http://www.npr.org/account/signup).
+
+The WordPress plugin is being developed as an Open Source plugin by NPR Digital Services. If you would like to suggest features or bug fixes, or better yet if you would like to contribute new features or bug fixes please contact Digital Services through our support page at [http://info.ds.npr.org/support.html](http://info.ds.npr.org/support.html).
+
+== Installation ==
+
+1. Upload the plugin files to the `/wp-content/plugins/plugin-name` directory, or install the plugin through the WordPress plugins screen directly.
+2. Activate the plugin through the 'Plugins' screen in WordPress
+3. Use the Settings->NPR API screen to configure the plugin. Begin by entering your API Key, then add your Push URL and Org ID. 
+
+
+== Frequently Asked Questions ==
+
+= Can anyone get an NPR API Key? =
+
+That's up to NPR but you can [register for an account here](http://www.npr.org/account/signup).
+
+= Can anyone push content into the NPR API using this plugin? =
+
+Push requires an Organization ID in the NPR API, which is typically given out to only NPR stations and approved content providers. If that's you, you probably already have an Organization ID.
+
+= Where can I find NPR's documentation on the NPR API? =
+
+The best resource for general information about the NPR API is here: [https://nprsupport.desk.com/customer/en/portal/topics/783619-api/articles](https://nprsupport.desk.com/customer/en/portal/topics/783619-api/articles).
+
+There's also the documentation in the NPR API site: [http://www.npr.org/api/index.php](http://www.npr.org/api/index.php).
+
+= Is there an easy way to directly query the NPR API? =
+
+You bet, just visit the NPR Query Generator: [http://www.npr.org/api/queryGenerator.php](http://www.npr.org/api/queryGenerator.php)
+
+== Screenshots ==
+
+NPR API Plugin Settings screen
+
+![NPR API Plugin Settings screen](/assets/npr-api-wp-plugin-settings.png)
+
+NPR API multiple get settings
+
+![NPR API multiple get settings](/assets/npr-api-wp-plugin-multiple-get-settings.png)
+
+Get NPR Stories link in the dashboard
+
+![Get NPR Stories link in the dashboard](/assets/get-npr-stories-link.png)
+
+Getting an NPR Story by Story ID
+
+![Getting NPR Stories by Story ID](/assets/get-npr-stories-link.png)
+
+NPR Stories having got gotten
+
+![NPR Stories having got gotten](/assets/npr-stories.png)
+
+
+== Changelog ==
+
```

Comment:
is our update 1.6? if so, changes should be captured here before we tag the new release